scripts/cbmc-viewer/storm.py

```diff
@@ -0,0 +1,191 @@
+"""The results of cbmc-storm."""
```

Comment:
What's cbmc-storm?

Reply:
Deleted all references to cbmc-storm tool that has not been sufficiently maintained.

"CBMC Storm, a python based tool that automates common CBMC usage. CBMC is a model checker for C programs. In cases when running CBMC in a large codebase such as an operating system is infeasible  CBMC Storm can be used to automate  CBMC runs on specific entry points of interest."